classes/Manufacturer.php

```diff
@@ -152,7 +152,7 @@ public function deleteSelection($selection)
         $result = true;
         foreach ($selection as $id) {
             $this->id = (int) $id;
-            $this->id_address = Manufacturer::getManufacturerAddress();
+            $this->id_address = self::getManufacturerAddress();
```

Comment:
Shouldn't we have `static`? This would be part of one of the next levels?

Reply:
Fixed